feedthefox/users/urls.py

```diff
@@ -3,7 +3,9 @@
 from django.conf.urls import include, url
 
 from allauth.account import views as account_views
-from allauth.socialaccount import providers, views as socialaccount_views
+from allauth.socialaccount import providers
+
+from feedthefox.users import views
```

Comment:
Why should we default to ``views`` and change the other one. If we are going to use different import name for views we need to be consistent.